database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -370,13 +387,11 @@ trait MetadataEntryComponent {
       ON me.#${callFqn} = avoidedCalls.#${callFqn}
       INNER JOIN #${wmse} wmse
       ON wmse.#${workflowUuid} = me.#${workflowUuid}
-      #${pgObjectInnerJoinStatement(isPostgres, metadataValue)}
       WHERE avoidedCalls.#${callFqn} IS NULL
-      AND (me.#${metadataKey} in ('executionStatus', 'backendStatus') AND #${dbMetadataValueColCheckName(isPostgres)} = 'Failed')
```

Comment:
Typing this comment out in the Files tab, not sure how it'll look in the Conversation tab. This was the problematic clause in the old query that essentially limited the attributes to just executionStatus and backendStatus. The query had to be updated so that a where clause could determine whether or not a task failed by something outside of the METADATA_ENTRY table values